libs/core/include/core/containers/mapping.hpp

```diff
@@ -1,111 +0,0 @@
-#pragma once
```

Comment:
Can we keep this please. This might not be useful currently, however, very useful in general

Reply:
yup